cs/filesystem.texy

```diff
@@ -24,6 +24,12 @@ delete(string $path): void .[method]
 Smaže soubor nebo adresář `$path`. Při chybě vyvolá výjimku `Nette\IOException`.
 
 
+move(string $source, string $destination, bool $overwrite=true): void .[method]
+-------------------------------------------------------------------------------
+
+Přesune soubor nebo adresář `$source` do cílového adresáře `$destination`. Ve výchozím nastavení přepisuje existující soubory a adresáře. S parametrem `$overwrite` nastaveným na hodnotou `false` vyvolá výjimku `Nette\InvalidStateException`, pokud cílový soubor nebo adresář `$destination` existuje. Při chybě vyvolá výjimku `Nette\IOException`.
```

Comment:
Ono to nepřesune do cílového adresáře, spíš to "přejmenuje adresář nebo soubor `$source` na `$destination`".